lib/nfc/protocols/mf_ultralight/mf_ultralight_listener.c

```diff
@@ -14,7 +14,15 @@ typedef enum {
     MfUltralightListenerAccessTypeWrite,
 } MfUltralightListenerAccessType;
 
-typedef bool (*MfUltralightListenerCommandCallback)(MfUltralightListener* instance, BitBuffer* buf);
+typedef enum {
+    MfUltralightCommandNotFoundStatus,
+    MfUltralightCommandProcessed,
+    MfUltralightCommandNotProcessedNAK,
+    MfUltralightCommandNotProcessedSilent,
+} MfUltralightCommandStatus;
```

Comment:
Let's keep naming as everywhere else:
```
typedef {
    TypeNameFirst,
    TypeNameSecond,
    ...
}TypeName;
```

Here I would delete "Status" and leave just MfUltralightCommand